samples/nrf9160/nrf_cloud_rest_cell_pos/README.rst

```diff
@@ -0,0 +1,98 @@
+The sample requires one of the following:
+* The device is provisioned and associated with an `nRF Cloud`_ account. The sample supports JITP provisioning via REST.
+* A private key is installed on the device and its associated public key is registered with an `nRF Cloud`_ account.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Overview
+********
+
+After the sample initializes and connects to the network, it will enter single-cell mode and send a single-cell location request to nRF Cloud using network data obtained from the ref:`Modem Info <modem_info_readme>` library.
+To enable multi-cell mode, press button 1.
+In multi-cell mode, the sample will request a neighbor cell measurement using the ref:`LTE Link Control <lte_lc_readme>` library.
+If neighbor cell data is measured, the sample will send a multi-cell location request to nRF Cloud. Otherwise, the request will be single-cell.
+In either mode, the sample will send a new location request if a change in cell ID is detected.
+
+See the `nRF Cloud Location Services documentation`_ for additional information.
+
+Limitations
+***********
+
+The sample requires the network carrier to provide date and time to the modem.
+Without a valid date and time, the modem cannot generate JWTs with an expiration time.
+
+User interface
+**************
+
+If you have the option :ref:`CONFIG_REST_CELL_POS_DO_JITP <CONFIG_REST_CELL_POS_DO_JITP>` enabled and you press button 1 when prompted at startup, it will request just-in-time provisioning (JITP) through REST with nRF Cloud.
+This is useful when initially provisioning and associating a device on nRF Cloud.
+You only need to do this once for each device.
+
+When you start up your device and have not requested JITP, the sample will ask if the location card on nrfcloud.com should be enabled in the device's shadow. This is only valid for provisioned devices.
+Press button 1 to perform the shadow update.
+The location card displays the device's location on a map and is not required for this sample to function.
+You only need to do this once for each device.
+
+After the sample completes any requested JITP or shadow updates, a button 1 press will toggle between single-cell and multi-cell mode.
+
+Configuration
+*************
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following configuration options for the sample:
+
+.. _CONFIG_REST_CELL_POS_DO_JITP:
+
+CONFIG_REST_CELL_POS_DO_JITP - Enable prompt to perform JITP via REST
+   This configuration option defines if the application will prompt the user for just-in-time provisioning on startup.
+
+.. _CONFIG_REST_CELL_POS_BUTTON_EVT_NUM:
+
+CONFIG_REST_CELL_POS_BUTTON_EVT_NUM - Button number
+   This configuration option defines the button to use for device interactions.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/nrf_cloud_rest_cell_pos`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
```

Comment:
Looking at the other READMEs' "Testing" section, it looks like for this README it would be a numbered list of what is already in the "Overview" section. 

I will do whatever you recommend.